app/controllers/arrivalP5/ViewAllArrivalsP5Controller.scala

```diff
@@ -87,7 +88,8 @@ class ViewAllArrivalsP5Controller @Inject() (
               href = controllers.arrivalP5.routes.ViewAllArrivalsP5Controller.onPageLoad(None, None).url,
               additionalParams = Seq(
                 searchParam.map("mrn" -> _)
-              ).flatten
+              ).flatten,
+              navigationHiddenText = Some(messages("arrival.notification.next.hidden.text"))
```

Comment:
Very minor but could just pass the message key in here and resolve it later with `messages(template, messages("govuk.pagination.previous")))` instead of `MessageFormat.format(template, messages("govuk.pagination.previous")))`